data/assets/scene/digitaluniverse/grids.asset

```diff
@@ -311,19 +306,20 @@ local plane10ly = {
     Rotation = {
       Type = "StaticRotation",
       Rotation = eclipticRotationMatrix
+    },
+    Scale = {
```

Comment:
Is this `StaticScale` really necessary?  If the grid is 1ly and the Size is 2*lightYear size a defined?